src/ui/TimerView.tsx

```diff
@@ -246,7 +246,7 @@ export class TimerView extends React.Component<Props, State> {
             toast.error(e);
         };
         this.connection.onmessage = (e) => {
-            // FIXME: Clone the Shared Timer. This assumes that `this` is always
+            // TODO: Clone the Shared Timer. This assumes that `this` is always
```

Comment:
I started using FIXME as long term things that need to be fixed and TODOs being temporary self reminders to do something as part of the ongoing PR. So unless you plan to fix this as part of this PR, this should stay a FIXME.

Reply:
In that case, can we rename our existing TODOs to FIXMEs instead?